src/article_metrics/utils.py

```diff
@@ -396,3 +398,31 @@ def _merge(a, b):
         c.update(copy.deepcopy(b))
         return c
     return reduce(_merge, dicts)
+
+def get_article_versions(article_id):
+    """
+    Fetches the versions of a given article based on the latest doiVersion.
+
+    Parameters:
+    article_id (str): The ID of the article for which versions are to be fetched.
+
+    Returns:
+    list: A list of integers representing the versions of the article.
+    If an error occurs, an empty list is returned.
+
+    Example:
+    >>> get_article_versions('85111')
+    [1, 2, 3]
+    """
+    try:
+        response = requests.get(f"{settings.LAX_URL}/{article_id}")
+        response.raise_for_status()
+        data = response.json()
+        doi_version = data.get('doiVersion')
+        if doi_version:
+            version = doi_version.split('.')[-1]
```

Comment:
minor: maybe would call this `latest_version_number` or similar. Just reading `version` makes me think it is a particular version and why we use range. But because as it is the latest version number, it is also the count.